cryptofeed/provider/whale_alert.py

```diff
@@ -0,0 +1,182 @@
+'''
+Copyright (C) 2017-2020  Bryant Moscon - bmoscon@gmail.com
+Please see the LICENSE file for the terms and conditions
+associated with this software.
+'''
+
+import asyncio
+import aiohttp
+import logging
+from time import time
+import json
+from json import JSONDecodeError
+
+from cryptofeed.defines import WHALE_ALERT, TRANSACTIONS
+from cryptofeed.feed import RestFeed
+from cryptofeed.standards import pair_exchange_to_std
+from cryptofeed.exceptions import RestResponseError
+
+
+LOG = logging.getLogger('feedhandler')
+to_from_data = ('address', 'owner_type', 'owner')
```

Comment:
can't you define this in the method its used in? why is this global?

Reply:
Yes, I can define it directly in the method using it.
Well, I am kind of always caring about CPU performance (always, like really always :)), and this data is absolutely not dynamic. So once the source is read, python has it, once and for all the execution of the code (this is my understanding).
If I implement it where it is used (in `_transactions()`), it would then be declared for every iteration, every coin, over and over again...
I will do as you propose Bryant, but I am curious to know what is the drawback at keeping it global within `whale_alert.py`?

Reply:
if you want it in the global scope then please make it capitalized, TO_FROM_DATA. Also, pretty sure it would be faster with a set than a generator for what you are doing. The iterable is only 3 elements, a generator is overkill

Reply:
Bryant!
It is not a generator, it is a tuple. A tuple is event faster to declare than a list.

Reply:
> if you want it in the global scope then please make it capitalized, TO_FROM_DATA.

Ok, doing right away!